packages/teams-js/package.json

```diff
@@ -20,6 +20,8 @@
     "test": "jest"
   },
   "dependencies": {
+    "@types/dom-mediacapture-transform": "0.1.2",
```

Comment:
I assume this is the main difference between this change and the previous version that broke a customer. Before we approve this, you will need to be able to explain to us why this change will fix that customer issue. Also, you will need to be able to reproduce the customer reported issue: https://github.com/OfficeDev/microsoft-teams-library-js/pull/1628

Once you can reproduce their issue, please verify that this fix will no longer break them.

Reply:
Reproduced the issue and confirmed the fix won't bring the same issue again, please see the comment I posted below.